exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -361,6 +361,24 @@ void nodeSubmittedTransactionsArePlacedInPool() throws Exception {
     }
   }
 
+  @Test
+  void getTransactionPool() throws Exception {
+    try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
+      TestService service = testKit.getService(TestService.SERVICE_ID, TestService.class);
+
+      TransactionMessage message = constructTestTransactionMessage("Test message", testKit);
+      RawTransaction rawTransaction = TestKit.toRawTransaction(message);
```

Comment:
I'd merge https://github.com/exonum/exonum-java-binding/pull/895 first and fix these

Reply:
Yes, that's a good catch, there isn't such a method in 895